src/egon/data/datasets/power_plants/pv_rooftop_buildings.py

```diff
@@ -2351,12 +2351,12 @@ def add_bus_ids_sq(
     """
     grid_districts_gdf = grid_districts(EPSG)
 
-    mask = buildings_gdf.scenario == "status_quo"
+    mask = buildings_gdf.scenario.str.contains("status")
```

Comment:
This is not the same check, it would also return true for all statusXXX scenarios, which it did not do before. Is that intended?

Reply:
If yes, `startswith` might be more appropriate...

Reply:
The function was not working in the way that it did not assigned any bus to any generator. This due to the filter [here](https://github.com/openego/eGon-data/blob/1daeaf6fd262722918b481388219a65debc7af53/src/egon/data/datasets/power_plants/pv_rooftop_buildings.py#L2354) that discarded all entries because any of them are equal to "status_quo". So the next lines of that function were applied over a data frame with no rows, without throwing an error.